src/seedu/addressbook/Main.java

```diff
@@ -127,5 +128,19 @@ private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorage
         return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
     }
 
-
+    /**
+     * @param result of the executed command from user
+     * if the result indicates to automatically list addressbook, this function
+     * 'creates' and executes a new list result.
+     * otherwise it does nothing
+     */
+    private void checkForAutomaticList(CommandResult result){
+        if (result.getIsAutomaticList() == true){
```

Comment:
Works, but let's keep the formatting consistent, and insert the missing spaces (e.g. `...checkForAutomaticList(CommandResult result) {`). Same for the next line!